docs/source/models.rst

```diff
@@ -124,32 +134,56 @@ class:
         (cm)", "type": "double"}, {"name": "petal length (cm)", "type": "double"}, {"name":
         "petal width (cm)", "type": "double"}]'
       outputs: '[{"type": "integer"}]'
+      
+Tensor-based Signature Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Only DL flavors support tensor-based signatures (i.e TensorFlow, Keras, PyTorch, Onnx, and Gluon).
```

Comment:
Isn't fastai a DL flavor?

Reply:
hm.. do we plan on supporting tensor input for fastai? @tomasatdatabricks